components/reddit/sources/new-hot-post-on-a-subreddit/new-hot-post-on-a-subreddit.js

```diff
@@ -0,0 +1,191 @@
+const reddit = require("../../reddit.app.js");
+module.exports = {
+  key: "new-hot-post-on-a-subreddit",
+  name: "New hot post on a subreddit",
```

Comment:
"posts" plural

Reply:
sorry if I missed that in my last comment

Reply:
got cha.